zeebe/exporters/camunda-exporter/src/test/java/io/camunda/CamundaExporterHandlerIT.java

```diff
@@ -407,7 +410,7 @@ void shouldExportUsingGroupCreateUpdateHandler(
       throws IOException {
     final var handler = getHandler(config, GroupCreatedUpdatedHandler.class);
     basicAssertWhereHandlerCreatesDefaultEntity(
-        handler, config, clientAdapter, defaultRecordGenerator(handler));
+        handler, config, clientAdapter, groupRecordGenerator(handler, GroupIntent.CREATED));
```

Comment:
🔧 Since the handler should handle both intent CREATED and UPDATED, wdyt about providing randomly one of them ? You can put the intent into a List and retrieve a single element by providing a random number

Reply:
I can also add another test for the `UPDATED` intent, that way they are both covered in all test runs.

Reply:
Yeah, that's even better 😄